app/src/main/java/io/appium/uiautomator2/server/ServerConfig.java

```diff
@@ -16,10 +16,131 @@
 
 package io.appium.uiautomator2.server;
 
+import java.util.HashMap;
+import java.util.Map;
+
+import io.appium.uiautomator2.model.settings.MjpegBilinearFiltering;
+import io.appium.uiautomator2.model.settings.MjpegScalingFactor;
+import io.appium.uiautomator2.model.settings.MjpegServerFramerate;
+import io.appium.uiautomator2.model.settings.MjpegServerPort;
+import io.appium.uiautomator2.model.settings.MjpegServerScreenshotQuality;
+import io.appium.uiautomator2.model.settings.ServerPort;
+
 public class ServerConfig {
-    private final static int PORT = 6790;
+    // server port (default: `6790`)
+    private final static int SERVER_PORT =
+        getValueFromEnvOrDefault("SERVER_PORT", 6790);
+    // mjpeg server port (default: `7810`)
+    private final static int MJPEG_SERVER_PORT =
+        getValueFromEnvOrDefault("MJPEG_SERVER_PORT", 7810);
+    // mjpeg fps should be between `1` and `60` (default: `10`)
+    private final static int MJPEG_SERVER_FRAMERATE =
+        getValueFromEnvOrDefault("MJPEG_SERVER_FRAMERATE", 10);
+    // mjpeg scale should be between `1` and `100` (default: `50`)
+    private final static int MJPEG_SCALING_FACTOR =
+        getValueFromEnvOrDefault("MJPEG_SCALING_FACTOR", 50);
+    // mjpeg quality is between 0 and 100 (default: `50`)
+    private final static int MJPEG_SERVER_SCREENSHOT_QUALITY =
+        getValueFromEnvOrDefault("MJPEG_SERVER_SCREENSHOT_QUALITY", 50);
+    // if filtering should be used in mjpeg resize operation (default: `false`)
+    private final static boolean MJPEG_BILINEAR_FILTERING =
+        Boolean.parseBoolean(System.getenv("MJPEG_BILINEAR_FILTERING"));
+
+    // In-memory overrides
+    private static Map<String, Object> overrides = new HashMap<>();
+
+    /* Helper Method(s) */
+
+    private static Integer getValueFromEnvOrDefault(String key, Integer defaultValue) {
+        String value = System.getenv(key);
+        if (value == null) {
```

Comment:
a ternary operator usage could be more readable here and below

Reply:
This will be resolved w/ the upcoming update(s)